### ISSUE TITLE
TypeScript complains about non-existent inherited static method

### ISSUE THREAD

**Issue by gbcreation** [xaviergonz/mobx-keystone#437]
Example code:

```javascript
@model('A')
class A extends Model({}) {
    static create() { return new A({}) }
}

@model('B')
class B extends ExtendedModel(A, {}) {}

B.create() // <= TypeScript error: Property 'create' does not exist on type 'typeof B'
```

**Comment by xaviergonz** [comment]
new B({})

**Comment by gbcreation** [comment]
Hi @xaviergonz ,
Sorry, my original example is somehow misleading. Actually, what the static method does doesn't really matter. The main point is that TypeScript complains about any inherited static method.

```typescript
@model('A')
class A extends Model({}) {
    static sayHello() { console.log('Hello!') }
}

@model('B')
class B extends ExtendedModel(A, {}) {}

B.sayHello() // <= TypeScript error: Property 'sayHello' does not exist on type 'typeof B'
```

**Comment by xaviergonz** [comment]
A.sayHello, statics are not inherited (that's not something particular for mobx keystone but for Javascript / TypeScript in general btw

**Comment by gbcreation** [comment]
This code seems to work correctly:

```javascript
class A {
    static sayHello() { console.log('Hello!') }
}

class B extends A {}

B.sayHello() // TypeScript does not complain about non-existent method and "Hello!" is displayed in the console
```

**Comment by xaviergonz** [comment]
ahh, my bad then. I'll see if that can be fixed

**Comment by xaviergonz** [comment]
@gbcreation fixed in 0.67.2

**Comment by gbcreation** [comment]
Thank you
